stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -945,6 +945,14 @@ func.func @broadcast_in_dim_unranked_operand(%arg0 : tensor<*xf32>) -> tensor<2x
 
 // -----
 
+// CHECK-LABEL: func @broadcast_in_dim_zero_rank
```

Comment:
Do we need this as a separate positive test?

Reply:
No, not necessary, I'll remove this.